pkg/console/operator/sync_v400.go

```diff
@@ -303,7 +303,17 @@ func (co *consoleOperator) SyncConfigMap(
 		useDefaultCAFile = false
 	}
 
-	defaultConfigmap, _, err := configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, managedConfig, infrastructureConfig, consoleRoute, useDefaultCAFile)
+	monitoringSharingConfig, mscErr := co.configMapClient.ConfigMaps(api.OpenShiftMonitoringNamespace).Get(api.SharingConfigMapName, metav1.GetOptions{})
+	if mscErr != nil && !apierrors.IsNotFound(mscErr) {
+		return nil, false, "FailedMonitoringSharingConfig", mscErr
```

Comment:
Shall we do `FailedGeMonitoringSharingConfig`? Its long, but the `GET` matches with other `Get` and `Apply` we have.  Though I see above we made an oops and just did `FailedManagedConfig` instead of `FailedGetManagedConfig`.